doc/tutorials/Compose-an-Interaction.md

```diff
@@ -1,6 +1,8 @@
-# Interaction Scripting with Pattern Player
+# Compose Interaction
 
-![packges](images/PatternBehaviorTree.png)
+_This example uses the sequential pattern player found in the harmoni_pattern package._
+
+![packges](../images/PatternBehaviorTree.png)
 
 _An example interaction script pictured as a behavior tree._
 
```

Comment:
I would suggest we state explicitly that the javascript example below is a template, and that further down is an actual demo example, so that people are not confused thinking that what they should make looks like the javascript. (Obvious for some, but maybe not for novice users).

Reply:
Definitely. This was copied from the old tutorial. We probably need to run through many of our tutorials again to make sure they're still good.